Library/Formula/libhttpserver.rb

```diff
@@ -0,0 +1,55 @@
+class Libhttpserver < Formula
+  homepage "https://github.com/etr/libhttpserver"
+  url "https://github.com/etr/libhttpserver/archive/v0.7.1.tar.gz"
+  head "https://github.com/etr/libhttpserver.git"
+  sha1 "b36c8a5f523c04613d6a36c86bbb31a2b92aab83"
+
+  option "with-cpp11", "Enable c++11 std classes"
+  depends_on "libmicrohttpd"
+
+  depends_on "pkg-config" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  option :universal
+  patch :p0, :DATA
+  def install
+    ENV.universal_binary if build.universal?
+
+    ENV["LIBMICROHTTPD_CFLAGS"] = `pkg-config libmicrohttpd --cflags`.chomp
+    ENV["LIBMICROHTTPD_LIBS"] = `pkg-config libmicrohttpd --libs`.chomp
+    args = [
+      "--disable-dependency-tracking",
+      "--disable-silent-rules",
+      "--prefix=#{prefix}"
+    ]
+
+    args << "--enable-cpp11" if build.with? "cpp11"
+
+    inreplace "bootstrap", "libtoolize", "glibtoolize"
+    system "./bootstrap"
+    mkdir "build" do
+      system "../configure", *args
+      system "make", "install"
+    end
+    prefix.install "examples"
+  end
+
+  test do
+    system ENV.cxx, "#{prefix}/examples/hello_world.cpp", "-o", "hello_world", "-lhttpserver"
```

Comment:
Eh…… it's an http server. If we run it, it will never exit……
it may be tested with a browser navigating to 127.0.0.1:8080, but i don't know how to do that.